sdks/java/extensions/timeseries/src/main/java/org/apache/beam/sdk/extensions/timeseries/io/tf/TFExampleToBytes.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.beam.sdk.extensions.timeseries.io.tf;
+
+import org.apache.beam.sdk.annotations.Experimental;
+import org.apache.beam.sdk.transforms.DoFn;
+import org.tensorflow.example.Example;
+
+/** Convert TensorFlow Example object to byte[]. */
+@Experimental
+public class TFExampleToBytes extends DoFn<Example, byte[]> {
```

Comment:
Can you share an example where you use these `TFExample*` `DoFns`? Beam elements are all technically byte arrays, and Beam has Coders to serialize and deserialize elements to/from bytes. And ultimately the IOs should understand how to do that. If these were some custom classes for which you needed to implement custom serialization for then it would probably make sense to implement custom coders. But these specific classes are subclasses of protobuf messages, which Beam should have good support for (e.g. there's `ProtoCoder` ([link](https://github.com/apache/beam/blob/279a05604b83a54e8e5a79e13d8761f94841f326/sdks/java/extensions/protobuf/src/main/java/org/apache/beam/sdk/extensions/protobuf/ProtoCoder.java)) that's used in GCP IOs, and there's `CoderUtils` [stuff](https://github.com/apache/beam/blob/279a05604b83a54e8e5a79e13d8761f94841f326/sdks/java/core/src/main/java/org/apache/beam/sdk/util/CoderUtils.java#L50) that can probably be used in conjunction). So my intuition is that it should be possible to use some generic solution that already exists in Beam to implement byte conversion, instead of creating custom classes for specific use case. Maybe I'm wrong here, and usage examples (and/or tests) would help understand this part